packages/jumpstarter-driver-corellium/jumpstarter_driver_corellium/driver.py

```
@@ -0,0 +1,180 @@
+            raise ValueError(f'Unable to fetch project: {self.parent.project_id}')
+        
+        # get instance and fail if instance does not exist
+        instance = self.parent.api.get_instance(self.parent.device_name)
+        if instance is None:
+            raise ValueError(f'Instance does not exist')
+
+        # destroy a virtual device
+        # wait until it's gone
+        self.parent.api.destroy_instance(instance)
+        while instance := self.parent.api.get_instance(instance) is not None:
+            selg.logger.info('Waiting for instance to be destroyed...')
```

Ruff (0.8.2)

Undefined name `selg`

(F821)